data/json/mapgen_palettes/shelter.json

```diff
@@ -238,38 +238,49 @@
     "id": "shelter_unfinished",
     "terrain": {
       " ": "t_region_groundcover_urban",
-      "!": "t_pavement",
-      "`": "t_pavement_y",
-      "&": "t_sidewalk",
       "#": "t_soil",
       "+": "t_door_frame",
       "-": "t_wall_wood",
+      "|": "t_wall_wood",
```

Comment:
I wonder if adding a low chance of using `t_wall_half` would make sense here, as an additional detail. Also applies to the line above.

Reply:
Actually, I should've used `t_wall_half` here since that's what I used in #58283